docs/getting_started/how_to_build_an_api.md

```diff
@@ -0,0 +1,215 @@
+# How to build an API
+
+The purpose of Gosoline is to help with API building. Much of its functionality is API related, and now we will cover the most important parts.
```

Comment:
I would be careful with such sentences as they might create the wrong impression. I would say that most of the code deals with things like database repositories, reading and writing to streams, or other stuff. Looking at the number of lines of code, the `apiserver` package just reaches 6.5k, not even as much as we have for ddb.